sov-modules/sov-modules-impl/integration-tests/tests/demo/basic_runtime.rs

```diff
@@ -191,14 +189,13 @@ fn check_query(storage: JmtStorage) -> bool {
 }
 
 use serial_test::serial;
-
 #[test]
 #[serial]
 fn test_demo_values_in_cache() {
     type C = MockContext;
-    delete_storage(DB_DIR);
 
-    let storage = JmtStorage::with_path(DB_DIR).unwrap();
+    let path = schemadb::temppath::TempPath::new();
+    let storage = JmtStorage::with_path(path).unwrap();
```

Comment:
nit: I don't think we re-use the path in this test, so we can just use `JmtStorage::temporary()`

Reply:
Good catch, fixed